File: URIOJ/uri1036.cpp
#include<cstdio>
#include<cmath>
int main(void){	
	double A, B, C, x1, x2, delta;
	while(scanf("%lf %lf %lf", &A, &B, &C) != EOF){
		delta = (B * B) - (4.0 * A * C);
		if (delta < 0.0 || A == 0.0)
			puts("Impossivel calcular");
		else{
			x1 = (-B + sqrt(delta)) / (2.0 * A);
			x2 = (-B - sqrt(delta)) / (2.0 * A);
			printf("R1 = %.5lf\nR2 = %.5lf\n", x1, x2);
		}
	}
	return 0;
}

File: URIOJ/uri1036.py
# -*- coding: utf-8 -*-
# Leonardo Deliyannis Constantin
# URI 1036 - Fórmula de Bhaskara

import sys

def raiz(nutella):
    return nutella ** 0.5

for line in sys.stdin:
    a, b, c = map(float, line.split())
    delta = b**2 - 4*a*c
    if a == 0.0 or delta < 0.0:
        print('Impossivel calcular')
        continue
    r1 = (-b + raiz(delta)) / (2*a)
    r2 = (-b - raiz(delta)) / (2*a)
    print('R1 = {:.5f}'.format(r1))
    print('R2 = {:.5f}'.format(r2))
